packages/loader/package.json

```diff
@@ -47,8 +47,17 @@
     "source-map": "^0.7.0"
   },
   "peerDependencies": {
+    "@rspack/core": "0.x || 1.x",
```

Comment:
We don’t need a peer dependency for every framework that’s compatible with Webpack.

```suggestion
```